src/obb.cpp

```diff
@@ -0,0 +1,206 @@
+#include <geometric_shapes/obb.h>
+
+#include <fcl/config.h>
+
+#if FCL_MAJOR_VERSION == 0 && FCL_MINOR_VERSION == 5
+#include <fcl/BV/OBB.h>
+typedef fcl::Vec3f FCL_Vec3;
+typedef fcl::OBB FCL_OBB;
+#else
+#include <fcl/math/bv/OBB.h>
+typedef fcl::Vector3d FCL_Vec3;
+typedef fcl::OBB<double> FCL_OBB;
+#endif
+
+namespace bodies
+{
+#if FCL_MAJOR_VERSION == 0 && FCL_MINOR_VERSION == 5
```

Comment:
Remove the FCL 0.5 checks for both `toFcl` and `fromFcl`. Or, even better, remove `toFcl` and `fromFcl` at all as they're no longer needed.